File: android/src/main/java/com/vgretailersdk/verfiyBankDetailsRequest.kt
package com.vgretailersdk
import android.util.Log
import com.google.gson.annotations.SerializedName

class VerifyBankDetailsRequest{
    @SerializedName("bankIfsc")
    var bankIfsc : String = ""
    @SerializedName("bankAccNo")
    var bankAccNo : String = ""
    @SerializedName("bankAccHolderName")
    var bankAccHolderName : String = ""
    @SerializedName("bankAccType")
    var bankAccType : String = ""
    @SerializedName("bankNameAndBranch")
    var bankNameAndBranch : String = ""
    @SerializedName("checkPhoto")
    var checkPhoto : String = ""
    
    constructor(bankIfsc: String, bankAccNo: String,bankAccHolderName: String,bankAccType: String,bankNameAndBranch: String,checkPhoto: String) {
        this.bankIfsc = bankIfsc
        this.bankAccNo = bankAccNo
        this.bankAccHolderName = bankAccHolderName
        this.bankAccType = bankAccType
        this.bankNameAndBranch = bankNameAndBranch
        this.checkPhoto = checkPhoto
    }

    fun printProperties(){
        Log.d("TAG","Properties of verifyBankDetailsRequest are :"+this.bankIfsc)
    }

}

File: ios/verfiyBankDetailsRequest.swift
import Foundation
class VerifyBankDetailsRequest: Codable {
    var bankIfsc: String?
    var bankAccNo: String?
    var bankAccHolderName: String?
    var bankAccType: String?
    var bankNameAndBranch: String?
    var checkPhoto: String?

    // Custom initializer to initialize the object with values from the NSDictionary
    init(details: NSDictionary) {
        bankIfsc = details["bankIfsc"] as? String
        bankAccNo = details["bankAccNo"] as? String
        bankAccHolderName = details["bankAccHolderName"] as? String
        bankAccType = details["bankAccType"] as? String
        bankNameAndBranch = details["bankNameAndBranch"] as? String
        checkPhoto = details["checkPhoto"] as? String
    }
}
